samples/nametranslation.d.ts.scala

```diff
@@ -16,7 +16,7 @@ object Nametranslation extends js.Object {
   val int8Array: js.typedarray.Int8Array = js.native
   val int16Array: js.typedarray.Int16Array = js.native
   val int32Array: js.typedarray.Int32Array = js.native
```

Comment:
Other likely candidates, if TypeScript has them: `ArrayBuffer`, `ArrayBufferView`, `DataView` and `Uint8ClampedArray`.

Reply:
Done